derive/src/lib.rs

```diff
@@ -111,19 +111,48 @@ impl Attributes {
     }
 
     /// Returns the pre and post visit token streams
-    fn visit(&self, s: TokenStream) -> (Option<TokenStream>, Option<TokenStream>) {
+    fn visit(&self, s: TokenStream, is_option: bool) -> (Option<TokenStream>, Option<TokenStream>) {
```

Comment:
Can you please also document what `is_option` means in this contex?

Reply:
The changes to the derive macro turned out not to be necessary